packages/web-frontend/public/index.html

```diff
@@ -150,6 +159,10 @@
     <noscript>
       <p style="color: white">You need to enable JavaScript to run this app.</p>
     </noscript>
-    <div id="root"><div class="spinner"></div></div>
+    <!-- App wide loading spinner.
+      It is placed in the index.html file so that it displays while the react app is mounting and it
+      is outside of the root so that it still displays while we fetch essential initial app data (eg. landing pages)-->
+    <div id="tupaia-spinner"></div>
```

Comment:
this feels a bit heavy handed but it was the only way that I could get the loader to continue seamlessly from the first render of the page right though until the landing page data is fetched.